### DISCUSSION TITLE
Contributing/Development guide for this repository?

### DISCUSSION THREAD

**Discussion by JoshuaKGoldberg** [vanilla-extract-css/vanilla-extract#1282] [Ideas]
I just sent #1281 - my first to vanilla-extract, yay! But I had to piece together a few parts of the development flow: installing, building, and changesets. They aren't documented anywhere I could find. Would it be possible to add a [standard CONTRIBUTING.md](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) file?

Steps I took to get the site running:

1. `pnpm i`
2. `pnpm build`
3. `cd site`
4. `pnpm start`

I'd be happy to send a PR, though I haven't actually built or tested anything outside `site/` directory. 😄

**Comment by askoufis** [comment]
Thanks for the reminder. It's on our todo list 👍.

**Comment by askoufis** [comment]
Added in https://github.com/vanilla-extract-css/vanilla-extract/pull/1769.
